### ISSUE TITLE
Spurious high velocities at corner walls with wedge6

### ISSUE THREAD

**Issue by spdomin** [NaluCFD/Nalu#248]
We have a few cases demonstrating high velocities at wedge elements when using wall functions.

Investigate and ensure exposed face quadrature points are proper (unit test, etc).

We can negotiate who takes this on... @rcknaus might be best with his master element unit test experience.

**Comment by mbarone81** [comment]
I was able to find high z-velocities, ~300 m/s occurring right near the blade tips. Elements are hex near the blade tip corners where these high velocities occur.

**Comment by spdomin** [comment]
@mbarone81, sounds like you are seeing something similar. As far as I can tell from our V27 simulation study, this issue may only occur when using wall functions at sharp corners. Your comment above suggests that the wall node is fine and that velocities are iffy away from surfaces.

In the end, your assessment of neat wall resolution is spot on. In the v27 case, the turbine box is where the issue is (at corners). However, cross wind is a low 6 m/s. As with most issues, there are probably a few things occurring that may contribute to the signs we are seeing.

**Comment by mbarone81** [comment]
My application tests with various element and mesh topologies at sharp corners indicated a sensitivity of the solution to resolution rather than to specific element types.  The wall function exacerbates the resolution issue because of thin shear layers that can appear at sharp corners.  This testing and the conclusions are documented in the ECP FY18 Q1 report.  The need for further unit testing of exposed face quadrature points may still be a good requirement to keep, however, I do not suspect a bug is the cause of the observed behaviors.

**Comment by spdomin** [comment]
Right... I think that you have explained the high velocity based on core studies that used the bug-fix in mapping.
